apps/gitness/src/pages-v2/pull-request/pull-request-conversation.tsx

```diff
@@ -95,23 +173,23 @@ export default function PullRequestConversationPage() {
   const [searchReviewers, setSearchReviewers] = useState('')
   const [addReviewerError, setAddReviewerError] = useState('')
   const [removeReviewerError, setRemoveReviewerError] = useState('')
-
   const [changesLoading, setChangesLoading] = useState(true)
   const [showDeleteBranchButton, setShowDeleteBranchButton] = useState(false)
   const [showRestoreBranchButton, setShowRestoreBranchButton] = useState(false)
+  const [rebaseErrorMessage, setRebaseErrorMessage] = useState<string | null>(null)
   const [errorMsg, setErrorMsg] = useState('')
+  const [comment, setComment] = useState<string>('')
+  const [isScrolledToComment, setIsScrolledToComment] = useState(false)
+
+  const [commentId] = useQueryState('commentId')
   const { spaceId, repoId } = useParams<PathParams>()
   const {
     scope: { accountId }
   } = useMFEContext()
-  const [comment, setComment] = useState<string>('')
-  const [commentId] = useQueryState('commentId')
-  const [isScrolledToComment, setIsScrolledToComment] = useState(false)
-
   const repoRef = useGetRepoRef()
   const { pullRequestId } = useParams<PathParams>()
-  const prId = (pullRequestId && Number(pullRequestId)) || -1
-  // const [loadState, setLoadState] = useState('data-loaded')
+
+  const prId = useMemo(() => (pullRequestId && Number(pullRequestId)) || -1, [pullRequestId])
```

Comment:
Probably no need to memoize this. The memo storage and look up will probably take longer than the action itself.